store/src/sqlite/interest.rs

```diff
@@ -119,6 +116,65 @@ where
             Err(err) => Err(err.into()),
         }
     }
+
+    async fn range_int(
+        &self,
+        left_fencepost: &Interest,
+        right_fencepost: &Interest,
+        offset: usize,
+        limit: usize,
+    ) -> Result<Box<dyn Iterator<Item = Interest> + Send + 'static>> {
+        let query = sqlx::query(
+            "
+        SELECT
+            key
+        FROM
+            interest_key
+        WHERE
+            key > ? AND key < ?
+        ORDER BY
+            key ASC
+        LIMIT
+            ?
+        OFFSET
+            ?;
+        ",
+        );
+        let rows = query
+            .bind(left_fencepost.as_bytes())
+            .bind(right_fencepost.as_bytes())
+            .bind(limit as i64)
+            .bind(offset as i64)
+            .fetch_all(self.pool.reader())
+            .await?;
+        //debug!(count = rows.len(), "rows");
+        Ok(Box::new(rows.into_iter().map(|row| {
+            let bytes: Vec<u8> = row.get(0);
+            Interest::from(bytes)
+        })))
+    }
+}
+
+#[async_trait::async_trait]
+impl<H> AccessInterestStore for InterestStore<H>
```

Comment:
I think it would help to use a qualified path for this trait. Something like `ceramic_api::AccessInterestStore` and then add a comment explaining why we expose the store to the API. This way its clear this is an explicit design decision for how the code all interacts.